src/main/scala/com/allantl/jira4s/v2/JiraMultiTenantClient.scala

```diff
@@ -5,7 +5,8 @@ import com.allantl.jira4s.v2.client._
 import com.softwaremill.sttp.SttpBackend
 
 sealed trait JiraMultiTenantClient[R[_]]
-    extends IssueClient[R, AuthContext]
+  extends IssueClient[R, AuthContext]
+    with WorkLogClient[R, AuthContext]
```

Comment:
`Worklog` instead of `WorkLog` to standardize with Jira Java Api. Can you change the rest as well?